ui/app/components/app-card.hbs

```diff
@@ -0,0 +1,39 @@
+{{!
+  ## Usage
+
+  <AppCard class="my-custom-class">
+    <:meta-primary>
+      v5
+    </:meta-primary>
+    <:meta-secondary>
+      Starting…
+    </:meta-secondary>
+    <:actions>
+      <a href="...">Visit</a>
+    </:actions>
+  </AppCard>
+}}
```

Comment:
thanks for adding an example comment for the named blocks 👍  (Unrelated but is there a preferred convention for the casing for those?)

Reply:
That’s a good question. [The RFC](https://github.com/emberjs/rfcs/blob/master/text/0460-yieldable-named-blocks.md) uses lowercase names for the blocks, but doesn’t specify anything strict.